pkgs/applications/office/timekeeper/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, poco
+, pkgconfig
+, gnome2
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "timekeeper";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "bburdette";
+    repo = "TimeKeeper";
+    rev = "363a3f0033fcca2d45b0d1f0fc74d6470c9db10d";
```

Comment:
Why not use a tag?

Reply:
tagged

Reply:
Using tags is preferred since due to a way GitHub is designed, a commit can refer to arbitrary fork of the repository, which makes it hard to review. Tags should not really change once created, otherwise an attacker could, for example, introduce a backdoor and change a tag to include it. Fortunately, the hash would change and notify us of something foul going on. Seriously, if there is a problem with the release, the project can just commit fixes and create new tag + 1.